content/iaed/1001-estruturas.sheet.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Estruturas de Dados (Cheat Sheet)
+description: Estruturas de Dados ;
```

Comment:
Change to a list of things in the cheat sheet:
`Lista Simplesmente Ligada, Pilha (Stack), Hash Table e AVL`